THEME.md

```diff
@@ -205,25 +205,46 @@ It's a good idea to add this to the top of the template to improve the styling o
 
 ## Action Injection
 
-The following will inject the action link (or button) into the e-mail:
+The following will inject the action link (or button) into the e-mail.
 
 ```html
 <!-- Action -->
+<% if(locals.action.constructor !== Array){ %>
```

Comment:
@eladnava I agree that it should be done at the top of the theme files. This was just a safeguard in case someone did not do that. For instance, I'm using someone else's theme that does not have this update but I want multiple buttons. Do you still think this should be removed?